src/database/queries/platformProviderResponses/insertOne.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO platform_provider_responses (
+  external_id,
+  platform_provider,
+  data
+)
+VALUES (
+  :externalId,
+  :platformProvider,
+  :data
+)
+ON CONFLICT (external_id, platform_provider) DO UPDATE
```

Comment:
Oh, I'm sorry, I'd meant to write this up for you - thanks for working it out, and I hope that didn't take too long!